files/configuration/pgadmin4.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+set -u
+
+
+# echo commands to terminal
+set -x
+
+
+#
+# pgAdmin 4 documentation: https://www.pgadmin.org/download/pgadmin-4-apt/
+#
+
+# install the public key for the repository
+curl -fsS https://www.pgadmin.org/static/packages_pgadmin_org.pub \
+    | sudo gpg --dearmor -o /usr/share/keyrings/packages-pgadmin-org.gpg
+
+# create apt configuration file
+sudo sh -c \
+'echo "deb [signed-by=/usr/share/keyrings/packages-pgadmin-org.gpg] \
+     https://ftp.postgresql.org/pub/pgadmin/pgadmin4/apt/$(lsb_release -cs) \
+     pgadmin4 main" > /etc/apt/sources.list.d/pgadmin4.list && apt update'
+
+# install pgAdmin 4
+sudo apt install pgadmin4-desktop
```

Comment:
`./uninstall.sh` will not work correctly – need to follow the existing paradigm for specifying packages (likely with some extensions to handle the case of pre-configuration).

Reply:
#25 is possibly related, but 33ef059 essentially resolves this issue

Reply:
Prompt when reinstalling 00-pgadmin4:

> File '/usr/share/keyrings/packages-pgadmin-org.gpg' exists. Overwrite? (y/N)

Reply:
Resolved by e365440